src/test/java/uk/gov/pay/adminusers/service/SecondFactorAuthenticatorTest.java

```diff
@@ -1,22 +1,37 @@
 package uk.gov.pay.adminusers.service;
 
+import com.warrenstrange.googleauth.GoogleAuthenticatorConfig;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import java.time.Clock;
+
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class SecondFactorAuthenticatorTest {
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
     private SecondFactorAuthenticator secondFactorAuthenticator;
+    private Clock clock;
+    private GoogleAuthenticatorConfig authConfig;
+    private long baseTime;
```

Comment:
Rather than a `long` of milliseconds since the epoch, I think an `Instant` would be a more appropriate type here. Additionally, a name like `initialTime` might be clearer.